packages/clerk-js/src/ui/common/InitialIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Text } from '../customizables';
+import { common } from '../styledSystem';
+
+type InitialIconProps = {
+  initials: string;
+};
+
+export const InitialIcon = ({ initials }: InitialIconProps) => {
+  return (
+    <Box
+      as='span'
+      sx={t => ({
+        ...common.centeredFlex('inline-flex'),
+        width: t.space.$4,
+        height: t.space.$4,
+        borderRadius: t.radii.$sm,
+        color: t.colors.$colorTextOnPrimaryBackground,
+        backgroundColor: t.colors.$primary500,
+      })}
+    >
+      <Text
+        as='span'
+        sx={t => ({
+          ...common.centeredFlex('inline-flex'),
+          width: '100%',
+          fontSize: t.fontSizes.$sm,
+          fontWeight: t.fontWeights.$semibold,
```

Comment:
We should be using a predefined text variant on the `Text` instead of directly `fontSize` and `fontWeight`.